Example/AccessibilitySnapshot/SwiftUITextEntry.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright 2024 Square Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#if swift(>=5.1) && canImport(SwiftUI)
```

Comment:
You can omit these checks now that our minimum Xcode version is 13.2.1

Reply:
This is done.